.changelog/1311.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:enhancement
+resource/cloudflare_logpush_job: add support for account-level logpush jobs
+resource/cloudflare_logpush_ownership_challenge: add support for account-level logpush ownership challenges
+```
```

Comment:
nearly there! for two entries, we need two explicit blocks.

~~~suggestion
```release-note:enhancement
resource/cloudflare_logpush_job: add support for account-level logpush jobs
```

```release-note:enhancement
resource/cloudflare_logpush_ownership_challenge: add support for account-level logpush ownership challenges
```
~~~

Reply:
updated.